### ISSUE TITLE
Batch Size

### ISSUE THREAD

**Issue by zneha** [premthomas/keras-image-classification#3]
Hello Prem, 
Eventhough you have set your batch size to be 32, under the training part, I see this:

Epoch 1/100
**5760/5760** [==============================] - 221s 38ms/step - loss: 1.4248 - categorical_accuracy: 0.5123 - val_loss: 10.7026 - val_categorical_accuracy: 0.3333

Shouldn't it be 32 instead on 5760? 5760 is all the training images right?
